src/main/java/net/minecraftforge/common/util/EnumHelper.java

```diff
@@ -141,6 +143,18 @@ public static EnumRarity addRarity(String name, TextFormatting color, String dis
     {
         return addEnum(EnumRarity.class, name, color, displayName);
     }
+    /**
+     * 
+     * @param name - The name the new enum constant should have
+     * @param textureLocation - The path to the texture for this armor type. Formatted as domain:path.
+     * @param armorStrength - How much protection this armor type should give
+     * @return The created HorseArmorType enum constant
```

Comment:
The wording is a bit awkward here. Maybe just say "The new {<span>@</span>code HorseArmorType}". Also, this method is `@Nullable`, which seems like a special case and should be documented in `@return`. [Reference (`@return`)](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@return)